src/skip-wip-ci/skip-wip-ci.yml

```diff
@@ -0,0 +1,62 @@
+# Orb Version 0.1.0
+
+version: 2.1
+description: 'Use to skip CI when PR title contains "wip", "skip ci", "ci skip" or is draft'
+
+commands:
+  check-skippable-pr:
+    steps:
+      - run:
+          shell: /bin/bash
+          name: Check Skippable PR
+          command: |
+            required_env_vars=(
+              "BUNDLE_GITHUB__COM"
```

Comment:
Do we usually have this `BUNDLE_GITHUB__COM` environment variable set across CI projects? Given how we're using it, I wonder if we should instead use a `GITHUB_TOKEN` environment variable. What do you think?

Reply:
We would have `BUNDLE_GITHUB__COM` present anytime a repo needs install private gems from github. That's true it's not every repo that needs that, so yeah i think a `GITHUB_TOKEN` var is good here.